src/main/resources/templates/appointment.html

```diff
@@ -7,185 +7,192 @@
-<nav class="navbar navbar-expand-lg navbar-light " style="background-color: #ffc629;">
-    <a href="#" class="pull-left"><img src="https://img.icons8.com/ios-filled/50/000000/caduceus.png" /></a>
-    <a class="navbar-brand" href="#">VirtuDoc</a>
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
-            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-    </button>
-
-    <div class="collapse navbar-collapse" id="navbarSupportedContent">
-        <ul class="navbar-nav ml-auto">
-            <li class="nav-item">
-                <a class="nav-link" href="#">Messages</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link" href="#">Reports</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link" href="#">Appointments</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link" href="#">Videos</a>
+
+<nav class="navbar navbar-expand-lg bg-warning navbar-dark py-1">
+    <div class="container-fluid">
+        <div class="col-md-1 text-center d-none d-md-block">
+            <img class="img-fluid" src="static/Doctors Symbol.png" />
+        </div>
+        <h1 ><a href="#home" class="navbar-brand mb-0 h1 fs-2">Virtudoc</a></h1>
+        <ul class="navbar-nav ms-auto">
+            <li>
+                <!-- Add navbar buttons here -->
+                <a href="#home" class="btn btn-info btn-outline-dark bg-light">Home</a>
             </li>
-            <button type="button" class="btn btn-light">Log Out</button>
         </ul>
     </div>
-
 </nav>
-    <!-- Hero image for experimentation. May change depending on
-    design on future sprint -->
 
-<section id="hero" class="d-flex align-items-center"></section>
-    <!-- END Hero -->
 
 
-    <!-- Begin Form Appointment with Header
-    Along with container for apptConfirmation page and form
-    for future sprint, .js script-->
 
-<h1>Schedule Your Appointment</h1>
+
+<h2> Book Appointment</h2>
+
+<h4> Let's Find Time on Your Calendar and Ours </h4>
+<div>
+    <p style="padding: 10px; border: 5px white;">
+        Here at VirtuDoc we strive to make it seamless to get in contact with
+        your selected physician, because your health is our top priority. Get in
+        contact with multiple different physicians that can provide the necessary
+        support to make sure you and your loved ones are taken care of. Looking
+        after your health should not be difficult. We aim to deliver. There are currently
+        three locations available in the Atlanta metropolitan area, including Marietta, Kennesaw,
+        and Buckhead.</p>
+</div>
 
 
-<div class="container">
-    <form action="apptConfirmation.html" method="get">
-        <label for="name">Full Name</label>
+<form action="AppointmentCreation.html" method="get">

```

Comment:
From our meeting on Sunday night, it sounds like you were already working on this, but this should probably be a POST request to help mitigate XSS attacks.